packages/atomic/tailwind.config.js

```diff
@@ -63,7 +63,9 @@ module.exports = {
   },
   variants: {
     extend: {
-      textColor: ['visited', 'group-focus'],
+      textColor: ['visited', 'group-focus', 'disabled'],
+      border: ['disabled'],
```

Comment:
What are those two for ? (border and cursor disabled)

Reply:
to be able to do `disable:text-primary` or `disable:border-primary` for example